Source/WebCore/Modules/permissions/PermissionStatus.h

```diff
@@ -28,10 +28,15 @@
 #include "ActiveDOMObject.h"
 #include "ClientOrigin.h"
 #include "EventTarget.h"
+#include "MainThreadPermissionObserverIdentifier.h"
+#include "Page.h"
 #include "PermissionDescriptor.h"
 #include "PermissionName.h"
 #include "PermissionObserver.h"
+#include "PermissionQuerySource.h"
 #include "PermissionState.h"
+#include "ScriptExecutionContextIdentifier.h"
+#include <atomic>
```

Comment:
What is this used for?

Reply:
There is another variable that uses atomic, but I suppose this can be removed.